src/plugins/auto/src/auto_compiled_model.cpp

```diff
@@ -26,7 +26,8 @@ AutoCompiledModel::AutoCompiledModel(const std::shared_ptr<ov::Model>& model,
 }
 
 void AutoCompiledModel::set_property(const ov::AnyMap& properties) {
-    OPENVINO_NOT_IMPLEMENTED;
+        OPENVINO_THROW_NOT_IMPLEMENTED("It's not possible to set property of an already compiled model. "
+                                       "Set property to Core::compile_model during compilation");
```

Comment:
@songbell don't we support changing device priorities in runtime?

Reply:
we supported only in auto ctput(multi) mode, but this seems to be deprecated. @peterchen-intel let's track if need to add it back.